doc/TBG_macros.cfg

```diff
@@ -188,6 +188,19 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# ISAAC server and rendering interval settings
+TBG_isaac="--isaac.period 1 --isaac.name !TBG_jobName --isaac.url <server_url>"
+
+# ISAAC JPEG quality (1..100). Smaller values are faster sent, but of lower quality
+TBG_isaac_quality="--isaac.quality 80"
+
+# ISAAC render resolution (default 1024x768)
+TBG_isaac_resolution="--isaac.width 1024 --isaac.height 768"
+
+# ISAAC direct pause option telling the ISAAC plugin to pause directly after
+# the start of the simulation
+TBG_isaac_pause="--isaac.direct_pause true"
```

Comment:
Well, I think this is not very consistent, because this only works for bool options with default value `false`. If an option is on per default you need to break your consistency using `--defaultFeature false` or to reverse the semantic somehow like `--noDefaultFeature`, which I myself don't like both. Or can any bool option shortend to the option name itself if it shall be set to true?